modules/ml/src/test/java/org/apache/ignite/ml/trainers/group/DistributedWorkersChainTest.java

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
Consider adding this and other tests in this package into an "umbrella" testsuite and adding that new testsuite to IgniteMLTestSuite